tests/test_heat.py

```diff
@@ -26,6 +38,17 @@ def test_heat_loss(self):
         np.testing.assert_array_less(demand, source)
 
     def test_zero_heat_loss(self):
+        """
+        This test is to check whehter the optimziation function when the zero heat Loss is used.
```

Comment:
I do not understand this sentence: "This test is to check whehter the optimziation function when the zero heat Loss is used." Do you mean: "Check the optimziation function when the zero heat loss is used" ?

Reply:
done